app/utils/common.ts

```diff
@@ -0,0 +1,37 @@
+import site from "~/config/site";
+
+/**
+ * Constructs the site title convention out of the current page title:
+ *
+ * ```ts
+ * constructSiteTitle("About") // "About | Philly JS Club"
+ * ```
+ * @param {string} pageTitle - The current page title
```

Comment:
[Docs] Type annotations such as `{string}` aren't generally necessary in JSDoc params. TypeScript does that for us already. So we can omit a bit of them:

```suggestion
 * @param pageTitle - The current page title
```

I've actually been working on this recently in linter-land and plan on bringing in some of the lint rules to this site eventually 😄 

* https://github.com/JoshuaKGoldberg/template-typescript-node-package/issues/303
* https://github.com/JoshuaKGoldberg/template-typescript-node-package/issues/371
* https://github.com/gajus/eslint-plugin-jsdoc/pull/1007

Reply:
Updated